kornia/enhance/shift_rgb.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+from kornia.testing import KORNIA_CHECK_IS_COLOR, KORNIA_CHECK_IS_TENSOR
+
+
+def shift_rgb(image, r_shift, g_shift, b_shift):
+    """Shift rgb channels.
+
+    Shift each image's channel by either r_shift for red, g_shift for green and b_shift for blue channels.
+    """
+
+    KORNIA_CHECK_IS_TENSOR(image)
+    KORNIA_CHECK_IS_COLOR(image, f"with shape {image.shape}")
+
+    shifts = [r_shift, g_shift, b_shift]
+
+    shifted = (image + torch.Tensor(shifts).view(1, 3, 1, 1).to(image)).clamp_(min=0, max=1)
+
+    return shifted
```

Comment:
So, what the types should be for those parameters?

```python
shift_rgb(image, r_shift, g_shift, b_shift):
```

Reply:
They are all torch.Tensor, I changed that

Reply:
Can `shifted = (image + torch.Tensor(shifts).view(1, 3, 1, 1).to(image)).clamp_(min=0, max=1)` be written in `torch` operations? Creating a new tensor here might lead to ONNX errors since it will loose track on the gradients from those 3 source tensors.

Reply:
It can be done like this then:

```
shifted = torch.empty_like(image)
for i, shift in enumerate(shifts):
    shifted[:, i, :, :] = (image[:, i, :, :] + shift).clamp_(min=0, max=1)
```

Reply:
Still a new tensor created. To avoid that, would `shifted = (image + torch.stack(shifts).view(1, 3, 1, 1).to(image)).clamp_(min=0, max=1)` work?

Reply:
The best way to know it is adding onnx and torchscript test

Reply:
It works with torch.stack

Reply:
Yes, but what do you mean by onnx conversion? Does this test have to create a PyTorch model and convert it to ONNX and then check if there will be any onnx errors because of RGB shifting?